### ISSUE TITLE
Widgets

### ISSUE THREAD

**Issue by Iliannnn** [JGeek00/adguard-home-manager#135]
### Description

It may be nice to have some widgets to put on your home screen. Maybe a widget with the server status, dns queries, blocked by filters etc

**Comment by JGeek00** [comment]
Unfortunately I think that's not possible. This application is created with Flutter, a multi platform framework. Widgets have to be written on native code for each platform, and I don't know how to do that.
